tests/src/esmvaltool/test_models.py

```diff
@@ -0,0 +1,65 @@
+from pathlib import Path
```

Comment:
I found the name of this file `test_models` to be quite confusing. Can we make it more pydantic-specific?

Reply:
I like schema too

Reply:
schema it is